### PR TITLE
Resolve the issue Cookie + Filter Control lost select option on refresh browser #4033

### PR DESCRIPTION
**Pull request by manukieli**
They resolve this issue

### CODE REVIEW

**Comment by UtechtDustin** [comment]
A Demo of the fix would be great (jsfiddle), thank you for very much the contribution.

**Comment by UtechtDustin** [comment]
Tested local, don't work for me, it just add a new option with the last selected item (duplicate of the item) and select it.
But the Table don't filter.

Please update your PR.

**Comment by manukieli** [comment]
For me is working well.

**Comment by djhvscf** [comment]
Testing it

File: src/extensions/cookie/bootstrap-table-cookie.js [hunk 1]
```diff
@@ -172,10 +172,17 @@ const UtilsCookie = {
 
         const applyCookieFilters = (element, filteredCookies) => {
           $(filteredCookies).each((i, cookie) => {
-            if (cookie.text !== '') {
-              $(element).val(cookie.text)
-              cachedFilters[cookie.field] = cookie.text
-            }
+              if (cookie.text !== '' && $(element).is('input')) {
```
Comment (djhvscf) [inline src/extensions/cookie/bootstrap-table-cookie.js]: Save the `$(element)` in a const variable, so you can use it in following lines

**Comment by djhvscf** [comment]
Please consider to retest this solution with the latest version of filter-control extension

**Pull request closed without merging**